tests/python/gpu/test_contrib_amp.py

```diff
@@ -300,10 +303,42 @@ def check_amp_convert_hybrid_block():
         params = converted_model.collect_params()
         assert params["stage2_unit1_conv2_weight"].dtype == np.float16
 
+
+    def check_amp_convert_bucketing_module():
+        model = train_model(context=mx.current_context())
+        result_model = amp.convert_bucketing_module(model)
+        val_sent = []
+        batch_size = 128
+        invalid_label = -1
+        num_sentence = 1000
+        buckets = [5, 10, 20, 30, 40]
+        len_vocab = 50
+
+        for _ in range(num_sentence):
+            len_sentence = randint(6, max(buckets)-1) # leave out the two last buckets empty
```

Comment:
Why are the last two buckets left empty?

Reply:
i think its just to reduce number of binds and make the test faster, I used the same data prep step as in the example.